go/chat/inboxsource.go

```diff
@@ -752,9 +766,10 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 	if conversationRemote.ReaderInfo == nil {
 		errMsg := "empty ReaderInfo from server?"
 		conversationLocal.Error = &chat1.ConversationErrorLocal{
-			Message:    errMsg,
-			RemoteConv: conversationRemote,
-			Permanent:  false,
+			Message:           errMsg,
```

Comment:
Might be time to make a `makeError` function for this guy, I'm afraid in one of these places we'll forget something.